myhdl/conversion/_verify.py

```diff
@@ -137,9 +137,9 @@ def __call__(self, func, *args, **kwargs):
 
         if isinstance(func, _Block):
             if hdl == "VHDL":
-                inst = func.convert(hdl='VHDL')
+                inst = func.convert(hdl='VHDL',*args, **kwargs)
```

Comment:
Correct, the _args_ are not in the callee's list, but as it is a _special_ argument Python lets us get away with it?

Reply:
My test aims to test not only the assignment of the init value in the reset case, I also want to check if the initial value assignment during signal declaration happens. Therefore the test calls verify with the inital_values parameter:
```
 inst = init_reset_tb()
 assert conversion.verify(inst,initial_values=True) == 0
````
This only works with my change in _verify.py.  No idea if this is the right way, I'm not really familiar yet with the internals of MyHDL. My naive assumption was, that conversion.verify should identically to <inst>.convert()

Only with this change the VHDL simulation output of the test case is identical to the MyHDL simulation. Otherwise the first two output lines of the VHDL version contain XXXXXXXX.

Reply:
I just revisited the MyHDL docs, the definition of convert is

```<block_instance>.convert(hdl='Verilog', **kwargs)```
@cfelton
I was not reading your comment careful enough, you were just refering to *args. Indeed this is not needed. 
```

        if isinstance(func, _Block):
            if hdl == "VHDL":
                inst = func.convert(hdl='VHDL', **kwargs)
            else:
                inst = func.convert(hdl='Verilog', **kwargs)
```
is sufficient to reach my goal. Does everybody agree, than I will change the PR.